frontend/public/components/_row-filter.scss

```diff
@@ -1,11 +1,31 @@
 $color-row-filter-border: $color-pf-black-300;
 $color-row-filter-border--active: $color-pf-blue-300;
 
+.clearfix:after {
```

Comment:
There's already a `clearfix` from Bootstrap. Can we use that? (Or are you modifying what it does?)

https://getbootstrap.com/docs/3.3/css/#helper-classes-clearfix

Reply:
I updated it to use flexbox rather than float (see commit 2).